Gopkg.lock

```diff
@@ -94,8 +117,13 @@
 [[projects]]
   branch = "master"
   name = "github.com/hashicorp/consul"
-  packages = ["api","lib/freeport","testutil","testutil/retry"]
-  revision = "610f3c86a089817b5bd5729a3b8c2db33a9ae2b0"
+  packages = [
+    "api",
+    "lib/freeport",
+    "testutil",
+    "testutil/retry"
+  ]
+  revision = "73e3252076f69a06386a98a528bb79fa43bd538e"
```

Comment:
I verified this is the only dep that actually changed version - all the other changes here are due to `go dep` formatting differences with the current version!